perfmetrics/scripts/fetch_metrics.py

```diff
@@ -0,0 +1,33 @@
+"""Executes fio_metrics.py and vm_metrics.py by passing appropriate arguments.
+"""
+import socket
+import sys
+import time
+from fio import fio_metrics
+from vm_metrics import vm_metrics
+
+START_TIME = 'start_time'
+END_TIME = 'end_time'
+INSTANCE = socket.gethostname()
+PERIOD = 120
+
+if __name__ == '__main__':
+  argv = sys.argv
+  if len(argv) != 2:
+    raise TypeError('Incorrect number of arguments.\n'
+                    'Usage: '
+                    'python3 fetch_metrics.py <fio output json filepath>')
+
+  fio_metrics_obj = fio_metrics.FioMetrics()
+  print('Getting fio metrics...')
+  temp = fio_metrics_obj.get_metrics(argv[1])
+  print('Waiting for 250 seconds for metrics to be updated on VM...')
```

Comment:
add a comment here why you are waiting for 250 sec

Reply:
According to the Google Cloud Monitoring docs, it takes up to 240 seconds for sampled data to be visible on the VM metrics graph. Therefore, I am waiting for 250 seconds to make sure the returned metrics are not empty.
Doc link: [https://cloud.google.com/monitoring/api/metrics_gcp#:~:text=Count%20of%20bytes%20received%20from%20the%20network.%20Sampled%20every%2060%20seconds.%20After%20sampling%2C%20data%20is%20not%20visible%20for%20up%20to%20240%20seconds.](url)

Reply:
done